tests/Rule/LatteTemplatesRule/PresenterWithoutModule/LatteTemplatesRuleForPresenterTest.php

```diff
@@ -27,6 +27,26 @@ public function testVariables(): void
                 5,
                 'default.latte',
             ],
+            [
```

Comment:
I would like to not report issues which user/developer can't fix from latte. These are some internal nette/latte errors right? We should fix them by modifying compiled template or ignore them.

Reply:
Itnis not latte internal but secondary errors caused by use of unkniwn (mixed) variable. It is imho expected to ocxur on level 9. Same errors would occur in php code when foreach iterates on mixed and you try access properties of mixed.

Reply:
ok then